Jenkinsfile

```diff
@@ -0,0 +1,130 @@
+library 'magic-butler-catalogue'
+
+def projectName = "eslint-config"
+def repo = "logdna/${projectName}"
+
+pipeline {
+  agent none
+
+  options {
+    timestamps()
+    ansiColor 'xterm'
+  }
+
+  stages {
+    stage('Test Suite') {
+      matrix {
+        axes {
+          axis {
+            name 'NODE_VERSION'
+            values '10', '12', '14'
+          }
+        }
+
+        agent {
+          docker {
+            image "us.gcr.io/logdna-k8s/node:${NODE_VERSION}-ci"
+          }
+        }
+
+        environment {
+          HOME = "."
```

Comment:
This should really be more like:

```
          NPM_CONFIG_CACHE = '.npm'
          NPM_CONFIG_USERCONFIG = '.npm/rc'
          SPAWN_WRAP_SHIM_ROOT = '.npm'
```

Reply:
In addition to `HOME`, or in place of?

Reply:
That broke the build.  Even with `HOME` still in there, it couldn't auth to the github package registry.

Reply:
`cannot create /tmp/eslint-config-4/.npm/rc: Directory nonexistent` is why